src/battle_script_commands.c

```diff
@@ -9963,6 +10034,7 @@ static u32 ChangeStatBuffs(s8 statValue, u32 statId, u32 flags, const u8 *BS_ptr
                 statValue = -1;
             else if (gBattleMons[gActiveBattler].statStages[statId] == 2 && statValue < -2)
                 statValue = -2;
+            
```

Comment:
```suggestion
```

I would just change this back to be consistent with most of the style in this portion.